flutter_vlc_player_platform_interface/lib/src/utils/options/vlc_audio_options.dart

```diff
@@ -6,7 +6,7 @@ class VlcAudioOptions {
   /// Enable/Disable time stretching audio
   /// This allows playing audio at lower or higher speed without affecting
   /// the audio pitch
-  static String audioTimeStretch(bool enable) {
+  static String audioTimeStretch({required bool enable}) {
```

Comment:
This is a breaking change. Please review all classes that are publicly exposed from the library and revert any such changes.

Let's ignore the lints on public interfaces for now.